src/pages/Stream/Views/Explore/StaticLogRow.tsx

```diff
@@ -15,7 +16,46 @@ type LogRowProps = {
 	isPinned?: boolean;
 };
 
+const CopyFieldValue = (props: { fieldValue: any }) => {
```

Comment:
why the fieldvalue is `any` ?

Reply:
the name doesn't feel like its a react component.

Reply:
parseLogData has a return type of any., which is then passed to this function

Reply:
Ok then.